Bump allowed `electric` version to 1.2.2

evanob opened the pull request.

evanob committed:
Bump allowed `electric` version to 1.2.2

magnetised commented:
Thanks @evanob  -- will look next week. This lib is due some attention -- we're very busy on the core app atm.

evanob commented:
Thanks @magnetised! I can see all the improvements coming on the core, and I don't want to miss out on that fun :)

magnetised commented:
@evanob Have started the more difficult task of compatibility with HEAD, which has a bunch of breaking changes that are worth including. I should have that out for next week assuming stuff on the main repo goes according to plan. Will let you know.

evanob commented:
thanks for the update @magnetised, that's great!

evanob committed:
Move Electric.Shapes.Api impl to Phoenix.Sync.Electric.ApiAdapter

This was one-shotted by Opus 4.5. I'm not sure this is the _right way_
to solve the problem, but it _seems_ to work!

evanob committed:
Fix warnings

evanob commented:
Bumping this, in case it slipped off your radar. The approach I've been taking doesn't feel quite right 😅
